user_docs/en/userGuide.t2t

```diff
@@ -894,6 +894,20 @@ A key command is provided to return to the original page containing the embedded
 | Move to containing browse mode document | NVDA+control+space | Moves the focus out of the current embedded object and into the document that contains it |
 %kc:endInclude
 
+++ Native Selection Mode ++[NativeSelectionMode]
+By default When selecting text with the `shift+arrow` keys in Browse Mode, a selection is only made within NVDA's Browse Mode representation of the document, and not within the application itself.
+This means that the selection is not visible on screen, and copying text with `control+c` will only copy NVDA's plain text representation of the content. I.e. formatting of tables, or whether something is a link etc will not be copied.
+However, NvDA has a Native Selection Mode which can be turned on in particular Browse Mode documents (so far only Mozilla Firefox) which causes the document's native selection to follow NVDA's browse mode selection.
+%kc:beginInclude
+|| Name | Key | Description |
+| Toggle Native Selection Mode on and off | NVDA+shift+f10 | Moves the focus out of the current embedded object and into the document that contains it |

```

Comment:
Note that this is a description for the script which moves out of embedded objects, and not for the one for toggling native selection (very likely a copy/paste error).

Reply:
```suggestion
| Toggle Native Selection Mode on and off | NVDA+shift+f10 | Toggles native selection mode on and off |
```